interactions.go

```diff
@@ -117,6 +117,16 @@ func (ic *InteractionCallback) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// ValidateToken validates verificationTokens
+func (ic InteractionCallback) ValidateToken(verificationTokens ...string) bool {
```

Comment:
In this case, token means verification token, but Slack has two tokens.(the other is signing token.)
So we want to use more specific name in order to prevent users from making a mistake.

Reply:
FYI:
Verification Token is deprecated.
https://api.slack.com/authentication/verifying-requests-from-slack#verifying-requests-from-slack-using-signing-secrets__verification-token-deprecation